crates/transaction/manifests/src/manifests_security_shield/manifests_securify_shield_securified_entity.rs

```diff
@@ -0,0 +1,95 @@
+#![allow(dead_code)]
+use crate::prelude::*;
+use std::ops::Deref;
+
+use profile_supporting_types::AnySecurifiedEntity;
+
+pub trait TransactionManifestSecurifySecurifiedEntity:
+    TransactionManifestSetRolaKey
+{
+    fn apply_security_shield_for_securified_entity(
+        securified_entity: AnySecurifiedEntity,
+        input: TransactionManifestApplySecurityShieldSecurifiedInput,
+    ) -> Result<TransactionManifest>;
+}
+
+impl TransactionManifestSecurifySecurifiedEntity for TransactionManifest {
+    fn apply_security_shield_for_securified_entity(
+        securified_entity: AnySecurifiedEntity,
+        input: TransactionManifestApplySecurityShieldSecurifiedInput,
+    ) -> Result<Self> {
+        let TransactionManifestApplySecurityShieldSecurifiedInput {
+            security_structure_of_factor_instances,
+            apply_shield_manifest_kind: kind,
+        } = input.clone();
+
+        let entity_address = securified_entity.entity.address();
+
+        // ACCESS_CONTROLLER_CREATE_PROOF_IDENT
+        let mut builder = TransactionManifest::produce_owner_badge(
+            ScryptoTransactionManifestBuilder::new(),
+            &securified_entity.entity,
+        );
+
+        let access_controller_address = securified_entity
+            .securified_entity_control
+            .access_controller_address;
+
+        let factors_and_time_input = &AccessControllerFactorsAndTimeInput::new(
+            &security_structure_of_factor_instances,
+        );
+
+        // INITIATE RECOVERY
+        let (init_method, init_input) =
+            kind.input_for_initialization(factors_and_time_input);
```

Comment:
Depending on selector of which of the six `TransactionManifest` flavours as selected by `kind: TransactionManifestApplySecurityShieldKind` we use different roles for **initialization of recovery** (and the correct concrete scrypto "Input"-type (Box dyn-ed))

For details see `transaction_manifest_apply_security_shield_kind.rs` which was added by this PR